src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,15 +1,10 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-
 import seedu.addressbook.common.Utils;
 import seedu.addressbook.data.exception.DuplicateDataException;
 
+import java.util.*;
```

Comment:
The coding standard for CS2103 specifies to import just the specific classes you need. If this happens automatically, you can change "code style -> Java -> imports" and change the setting